### PR TITLE
Support for multi-line text using file as input and multiple text files too.

### PR DESCRIPTION
**Pull request by nitincodery**

### CODE REVIEW

**Comment by albertopasqualetto** [comment]
Hello, thanks for your PR!
Why should people select a file to input the text since the other parts of the program is interactive? Did you made this in order to satisfy your specific needs?
I was thinking to use [Editor type in inquirer](https://github.com/SBoudrias/Inquirer.js#editor---type-editor) (I do not know if you can use a new line character like `\n`.

Also I have the plan to pass inputs from command line at launch, in order to skip that specific section of the program.

What do you think?

**Comment by nitincodery** [comment]
I used file input because wasn't able to add multiline input in cli, also \n wasn't working in windows cmd so as ^ wasn't either, so I had input of multiple lines and also multiple messages so I first used two instances to send two different messages and sent everything in single line, then I thought if numbers are read from file why not messages can't be and I edited the code to take message as input and then allowing multiple files also allowed me to send multiple text message at once.

Also, I haven't used inquirer before and the Editor type looks promising.

I didn't get the part: what do you mean by passing inputs at launch, like does it mean passing parameters for example `sendBulk folder/numbers.txt msg1.txt media.png`, what inputs, I didn't get this part. If it does mean that, then how one will add multiple media files to command line parameter.

Also, I think I can do the part of using a folder as input to media files. I think I can do it, haven't started it yet. Also I was thinking of a notification to the bulk sender person on his whatsapp number about certain stats, which he can define in cli interface as optional input, and also if the program exits with an error save the last successful sent number in a file and resume the sending, because yesterday I used it and I re-run the program 8 times and manually had to get the last number it sent from whatsapp app, the executable exits when it is done or any network issue occurs at my end, later I ran it via node instead of executable to get the last successful sent, so I think I can add a resume functionality too.

Also, I don't know about compiling with local chromium in the release, I didn't used it, also I'm beginner so I don't know why it was added with it, must be something efficiency related aspect.

In short, two more things I thought I could add to it, i.e. resume from the last number it failed and a media folder support and about using file inputs for message was to use a multiline input because its confusing in different terminals to use a new line character, also if someone has pre-defined input from somewhere it has to be edited with a new line character and paste and I actually tried that too, I don't know for what reason it was enter-ing itself right after first line and was never able to use a multiline text, maybe I was missing something. But using a file eased it a lot for me.

And really very good project, kudos to you @albertopasqualetto 😄

**Comment by albertopasqualetto** [comment]
Thanks for kudos @nitincodery !

I think that this solution cannot be definitive, so I won't merge it, instead I will try implementing editor as I said and I created an [issue on inquirer's github asking for the new line in input type](https://github.com/SBoudrias/Inquirer.js/issues/1119).
Also I want move to your implementation to optional: with flag `-t <txtFile>` enable your implementation and disable interactive message input.

As per the incomprehension, I meant what you said, but with flags like `sendBulk -n "folder/numbers.txt" -t "msg1.txt" -f "media1.png" "media2.png"`, I think that this can be achieved with [commander's variadic options](https://github.com/tj/commander.js#variadic-option).

Then thank you so much for offering to implement the folder sending, I think I will add other TODO's for other ideas I have (if you have more, open an issue please).

The local chromium is necessary if you have not installed Google Chrome in your system, otherway the program won't work.
Also, Chrome (and not Chromium) is necessary if you want to send videos as media (I don't know and I did not test if it works also with Microsoft Edge (that one based on chromium)).
While writing I am also thinking that local chromium is not necessary neither if Chromium is installed, which is more popular on linux systems: I will add a to do...

For the log you are completely right, in my opinion a good idea could be creating a function which both does `console.log(...)` and writes to log.txt in the same folder of index, and then substituting all current `console.log` with that.

Finally, I don't get why stats should be sent to your personal number, could you please tell me a scenario example?

Thanks again a lot!

**Review by albertopasqualetto — APPROVED** [review]

**Comment by nitincodery** [comment]
Yeah I understood using flags, I couldn't get the language before, now it's clear and yes it's better idea to use message with optional text file flag.

For local chrome, I got it why it is needed. I need to learn to bundle it in release with index executable.

For the log sent to my personal number (is the same number used to authenticate whatsapp) is because I was thinking to release a arm executable and try executing it in termux android so that one can directly bulk send via phone and get the logs on his own number on whatsapp. I need to test the arm release wether it work or not on android inside [termux](https://f-droid.org/en/packages/com.termux/) application.

**Comment by albertopasqualetto** [comment]
Unfortunately the app won't work on termux because it is based on puppeteer which need a desktop browser; as far as I know you should download a linux distro in a container in termux, so start it in order to have a full browser and make this program work.

local chromium is basically the same local chromium folder in puppeteer module folder copied in the same folder of index.exe.

**Comment by albertopasqualetto** [comment]
I will close this PR, I added your contribute in README.md in #2
